<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Universal Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="library_hLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="LibraryPath">
          <property name="text">
           <string>./libTestLibrary.dll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="LibrarySelector">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>选择动态库</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="config_hLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="ConfigPath">
          <property name="text">
           <string>../TestConfig.json</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ConfigSelector">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>选择调用配置文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="main_vLayout" stretch="1,100">
      <item>
       <layout class="QHBoxLayout" name="load_hLayout">
        <item>
         <widget class="QPushButton" name="pB_LOAD">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>121</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTableWidget" name="param_inputTable"/>
        <widget class="QTextBrowser" name="result"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="test_gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pB_Test2">
        <property name="text">
         <string>Test2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pB_Test3">
        <property name="text">
         <string>Test3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pB_Test1">
        <property name="text">
         <string>Test1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="Library">
    <property name="title">
     <string>动态库</string>
    </property>
    <addaction name="SelectLibrary"/>
    <addaction name="Close"/>
   </widget>
   <widget class="QMenu" name="Config">
    <property name="title">
     <string>配置文件</string>
    </property>
    <addaction name="SelectConfig"/>
    <addaction name="Save"/>
    <addaction name="SaveAs"/>
   </widget>
   <widget class="QMenu" name="setting">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="recvEncode">
     <property name="title">
      <string>接收编码</string>
     </property>
     <addaction name="reUTF8"/>
     <addaction name="reGBK"/>
    </widget>
    <widget class="QMenu" name="outputEncode">
     <property name="title">
      <string>输出编码
</string>
     </property>
     <addaction name="oeUTF8"/>
     <addaction name="oeGBK"/>
    </widget>
    <addaction name="outputEncode"/>
    <addaction name="recvEncode"/>
   </widget>
   <addaction name="Library"/>
   <addaction name="Config"/>
   <addaction name="setting"/>
  </widget>
  <action name="SelectLibrary">
   <property name="text">
    <string>选择动态库</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="Close">
   <property name="text">
    <string>关闭动态库</string>
   </property>
  </action>
  <action name="SelectConfig">
   <property name="text">
    <string>选择配置文件</string>
   </property>
  </action>
  <action name="reUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="oeUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="oeGBK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GB18030</string>
   </property>
  </action>
  <action name="reGBK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GB18030</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
